api/v1beta1/backuppolicy_types.go

```diff
@@ -33,7 +34,7 @@ type BackupPolicySpec struct {
 	Schedule string `json:"schedule"`
 
 	// Specifies parameters for backup Pod.
-	JobConfig JobConfig `json:"jobConfig"`
+	JobConfig mocov1beta2.JobConfig `json:"jobConfig"`
```

Comment:
Using v1beta2 types to define v1beta1 types seems like a bug to me.

Reply:
I did this because BackupPolicy exists in the v1beta1 package and I wanted it to use a common struct with MySQLCluster in v1beta2.
Fixed to use after conversion.
https://github.com/cybozu-go/moco/pull/350/commits/4f5c0ae1595d2b591397a17b7fd8ac68133421cc